src/ProductView/FilterBar/FilterBar.js

```diff
@@ -1,14 +1,23 @@
 import React, { Component } from 'react';
-import InputField from '../../Utils/InputField/InputField.js';
+import RemovableInputField from '../../Utils/RemovableInputField.js';
+
 
 class FilterBar extends Component {
+    decideOnRemoveButton(currentFilter) {
+        if(this.props.filter[this.props.filter.length - 1] === currentFilter) {
+            return false;
+        } else {
+            return true;
+        }
+    }
+
     render() {
         return (
             <div className="filter-bar container">
                 <div className="form-group row">
                     {this.props.filter.map((filter) =>
-                        <InputField key={filter.id} id={filter.id} value={filter.value} placeholder="Search..."
-                                    onInputChange={this.props.onInputChange}/>
+                        <RemovableInputField key={filter.id} id={filter.id} value={filter.value} placeholder="Search..."
```

Comment:
Please break this component initialisation after each parameter.
Corresponding convention: If a component initialisation has more than 2 parameters, insert a line break after each parameter.
(I'll add the development conventions to the readme.)